mlxtend/plotting/__init__.py

```diff
@@ -17,6 +17,7 @@
 from .checkerboard import checkerboard_plot
 from .ecdf import ecdf
 from .scatterplotmatrix import scatterplotmatrix
+from .corr_circle_pca import plot_pca_correlation_graph
```

Comment:
The file could be named `pca_correlation_graph.py` instead of `corr_circle_pca.py`, which could make it slightly easier to find & relate to the imported function